packages/frontend/src/components/UserSettings/UsersAndGroupsPanel/InvitesModal.tsx

```diff
@@ -79,45 +79,38 @@ const InvitesModal: FC<{
                     <Group
                         spacing="xs"
                         align={form.errors.email ? 'center' : 'end'}
+                        noWrap
                     >
                         <TextInput
                             name="email"
                             label="Enter user email address"
                             placeholder="example@gmail.com"
                             required
                             disabled={isLoading}
-                            w="43%"
+                            style={{ flex: 1 }}
                             {...form.getInputProps('email')}
                         />
-                        <Group
-                            spacing="xs"
-                            align={form.errors.email ? 'center' : 'end'}
```

Comment:
Small issue when there is an error message.
<img width="643" height="187" alt="Screenshot 2025-07-29 at 13 51 21" src="https://github.com/user-attachments/assets/02ab9c2e-ea52-4519-95c3-ab303c85e7f3" />

Reply:
fixed!